components/tyk-dev-portal/values.yaml

```
@@ -11,46 +15,46 @@ global:
     # - adminUserPassword sets .global.adminUser.password
     useSecretName: ""
 
-    # email of Enterprise Portal admin
+    # email of Tyk Portal admin
     email: default@example.com
-    # password of Enterprise Portal admin
+    # password of Tyk Portal admin
     password: 123456
   tls:
     dashboard: false
-    enterprisePortal: false
+    devPortal: false
   servicePorts:
     dashboard: 3000
   bootstrap:
-    # setting this to true bootstraps Tyk Enterprise Portal.
-    enterprisePortal: false
+    # setting this to true bootstraps Tyk Developer Portal.
+    devPortal: false
 
   secrets:
-    # enterprisePortal secret stands for secret including Tyk Dashboard credentials.
+    # devPortal secret stands for secret including Tyk Dashboard credentials.
     # It should include the followings:
     # - TYK_ORG: Tyk Dashboard Organisation ID
     # - TYK_AUTH: Tyk Dashboard API Access Credentials
-    enterprisePortal: tyk-enterprise-portal-conf
+    devPortal: tyk-dev-portal-conf
 
-    # useSecretName can be used if you don't want to store plaintext values for Enterprise Portal license in
+    # useSecretName can be used if you don't want to store plaintext values for Tyk Portal license in
     # the Helm value file and would rather provide the k8s Secret externally.
     # You should set following fields in the secret
-    # - EnterprisePortalLicense - Sets LicenseKey for Enterprise Portal
-    # - EnterprisePortalStorageConnectionString - Sets Database.ConnectionString for Enterprise Portal
+    # - DeveloperPortalLicense - Sets LicenseKey for Tyk Portal
```

Since this key `DeveloperPortalLicense` stands for the license for new Portal, what do you think updating the comment, like the following:
```suggestion
    # - DeveloperPortalLicense - Sets LicenseKey for Tyk Developer Portal
```

Same for other fields' comments on this file